ecs-cli/modules/cli/compose/entity/entity_helper.go

```diff
@@ -74,6 +71,27 @@ func GetOrCreateTaskDefinition(entity ProjectEntity) (*ecs.TaskDefinition, error
 	return resp, nil
 }
 
+func createRegisterTaskDefinitionRequest(taskDefinition *ecs.TaskDefinition) *ecs.RegisterTaskDefinitionInput {
+	// Valid values for network mode are none, host or bridge. If no value
+	// is passed for network mode, ECS will set it to 'bridge' on most
+	// platforms, but Windows has different network modes. Passing nil allows ECS
+	// to do the right thing for each platform.
+	request := &ecs.RegisterTaskDefinitionInput{
+		Family:               taskDefinition.Family,
+		ContainerDefinitions: taskDefinition.ContainerDefinitions,
+		Volumes:              taskDefinition.Volumes,
+		TaskRoleArn:          taskDefinition.TaskRoleArn,
+	}
+
+	if networkMode := taskDefinition.NetworkMode; networkMode != nil {
+		if *networkMode != "" {
```

Comment:
I know this is annoying and doesn't fit with go code normally looks, but when using the AWS SDK Go you are not supposed to directly dereference points. Instead you're supposed to use `aws.StringValue`. (Use `aws.String` to create a pointer).